### DISCUSSION TITLE
Plugin to overrides Core features

### DISCUSSION THREAD

**Discussion by bishu** [ryanhowdy/fcms#648] [Ideas]
As of now I am using FCMS ver 3.7.0 and I have made many changes as per my requirement.
But when the FCMS ver 3.8.0 published then I have make all the changes again which will take lots of time again.

So, I have hope we should have a Plugin or Modules option which can override the FCS Core features. By this way we can save a lots of time after upgrading to newer version.

Then all the webmaster of FCMS can create their own Plugin as per their requirement without touching the hard core of the FCMS.

**Comment by ryanhowdy** [comment]
I'd love to know what additions/modifications you have made. Maybe they can be incorporated into the codebase.
